tests/plugins/api/test_default_definition_body_plugin.py

```diff
@@ -0,0 +1,51 @@
+from unittest import TestCase
+from mock import Mock, patch
+
+from samtranslator.plugins.api.default_definition_body_plugin import DefaultDefinitionBodyPlugin
+from samtranslator.public.plugins import BasePlugin
+
+IMPLICIT_API_LOGICAL_ID = "ServerlessRestApi"
+
+
+class TestDefaultDefinitionBodyPlugin_init(TestCase):
+
+    def setUp(self):
+        self.plugin = DefaultDefinitionBodyPlugin()
+
+    def test_plugin_must_setup_correct_name(self):
+        # Name is the class name
+        expected_name = "DefaultDefinitionBodyPlugin"
+
+        self.assertEquals(self.plugin.name, expected_name)
+
+    def test_plugin_must_be_instance_of_base_plugin_class(self):
+        self.assertTrue(isinstance(self.plugin, BasePlugin))
+
+
+class TestDefaultDefinitionBodyPlugin_on_before_transform_template(TestCase):
```

Comment:
I think this (and your test updates in general) misses the case where a ```DefinitionBody``` or ```DefinitionUri``` exists in part of a template but doesn't in another.  Tests exist that target the old SAM requirements (forcing either a ```DefinitionBody``` or ```DefinitionUri``` to be present) and you wrote new tests for the case where neither one exists at all. It might be worth it to have a test where both cases are present to enforce the desired behavior.